src/simulate.jl

```diff
@@ -27,9 +27,9 @@ function simulate(cp::AbstractNeuralNetworkControlProblem, args...; kwargs...)
     st_vars = state_vars(cp)
     inpt_vars = input_vars(cp)
     n = length(st_vars)
-    X₀ = Projection(initial_state(ivp), st_vars)
+    X₀ = project(initial_state(ivp), st_vars)
     if !isempty(inpt_vars)
-        W₀ = Projection(initial_state(ivp), inpt_vars) |> overapproximate
+        W₀ = project(initial_state(ivp), inpt_vars)
```

Comment:
I did not check if this works with nondeterministic inputs because we currently do not have such a model in the repo. I think there was the Unicycle model that @SebastianGuadalupe had in his notebook. Could you send a PR for that?

Reply:
i'd expect this is fine given the new https://github.com/JuliaReach/LazySets.jl/pull/2597 and also that such inputs  are usually hyperrectangular